self-core-impl/src/main/java/com/selfxdsd/core/StoredApiTokens.java

```diff
@@ -0,0 +1,48 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.ApiToken;
+import com.selfxdsd.api.ApiTokens;
+import com.selfxdsd.api.User;
+import java.util.*;
+
+/**
+ * Stored API Tokens.
+ */
+public final class StoredApiTokens implements ApiTokens {
```

Comment:
@andreoss You can remove this class, it's not needed. 

The base plural (which represents *all* the entities of a certain type) is always implemented in self-storage.

Here in the core, we have mocks for these plurals, the ``InMemory*`` classes.

But we are not implementing these InMemory* classes anymore because we already have the architecture and self-storage in place. They were useful in the beginning of the project, they were acting as some sort of scaffolding.

Reply:
@amihaiemil Done.